unfetter-discover/unfetter-ui
# Close reports

supports unfetter-discover/unfetter#557
lock a work product, upgrade angular to latest 4.4.6, better icons
fix file upload error messages
requires https://github.com/unfetter-discover/unfetter-store/pull/68

## To Test
* visit a threat report https://localhost:9443/#/threat-dashboard
* collapse the side panel, verify the main panel expands correctly
* select the lock icon, refresh page, notice state persists

* click thru the app, verify no angular lib upgrade regressions

* visit, https://localhost:9443/#/threat-dashboard/create
* click upload csv, select a bad file, verify an error message
